### ISSUE TITLE
RDLC Layout not found in Rendering with Caption

### ISSUE THREAD

**Issue by pri-kise** [wbrakowski/AL-Navigator#141]
I wanted to try this new feature, but it didn't found the RDLC file.

AL Snippet to test:
```al
    rendering
    {
        layout("Purchase Order (RDLC)")
        {
            Caption = 'Purchase Order (RDLC)';
            LayoutFile = './src/Purchase/Document/PurchaseOrder.rdlc';
            Summary = 'Detailed layout with all fields.';
            Type = RDLC;
        }
    }

```

This could help solving the issue.
```
layout\(['"].+?['"]\)\s*{\s*(Caption.+\s*)?LayoutFile\s*=\s*['"](.+?)['"]
```

**Comment by wbrakowski** [comment]
Hey @pri-kise, thank you for reporting this. I will have a look at it.

**Comment by wbrakowski** [comment]
This will be fixed with version 0.7.0. Should be available by evening.
Thank you again and have a nice day.
